test/images/test_image.py

```diff
@@ -63,6 +63,41 @@ def test_create_thumbnail(self):
         # the reason only 0s are being asserted is because this is checking the hash of
         # a purely black 300x300 square created in the test_image above
 
+    @pytest.mark.parametrize(
+        "image_size, config_thumbnail_size, expected_thumbnail_size",
+        [
+            pytest.param((300, 300), (50, 50), (50, 50), id="50x50 thumbnail"),
+            pytest.param((400, 300), (60, 80), (60, 80), id="60x80 thumbnail"),
+            pytest.param(
+                (300, 300),
+                (75, 100),
+                (75, 75),
+                id="75x100 thumbnail (square image)",
```

Comment:
This should probably be labelled '75x75 thumbnail (square image)'. The currently labelling confused me a bit.

Reply:
Good point, on reflection I can see how that might be confusing. Changed in 74992688d4bd21ab0f4e52ad18e8689493a84244